pubsub/kafka/kafka.go

```diff
@@ -368,3 +367,9 @@ func (k *Kafka) Close() error {
 func (k *Kafka) Features() []pubsub.Feature {
 	return nil
 }
+
+type asBase64String []byte
```

Comment:
This is overkill. Just invoke the one-liner directly: `base64.StdEncoding.EncodeToString(s)`

Reply:
See above